src/client/dfs/dfs.c

```diff
@@ -1867,7 +1867,7 @@ dfs_cont_create(daos_handle_t poh, uuid_t *cuuid, dfs_attr_t *attr,
 
 	/** Add root object */
 	entry.oid = roots.cr_oids[1];
-	entry.mode = S_IFDIR | 0755;
+	entry.mode = S_IFDIR | 0700; /* TODO: Should this use umask? */
```

Comment:
this change sounds totally unrelated to the PR.
if you want to keep it in this PR, please repush the PR with:
Feature: dfs pragma

Reply:
It's related but does not belong in this PR.  I filed the ticket as https://daosio.atlassian.net/browse/DAOS-12692 about this.